pip/main.ts

```diff
@@ -110,6 +134,8 @@ async function checkDryRun() {
     console.error(
       `[${red("phylum")}] The operation caused a threshold failure.\n`,
     );
+
+    logIssues(jobStatus);
```

Comment:
Why was this included in this patch? To me it seems like an unrelated feature which should go into a different PR.

Reply:
see: https://github.com/phylum-dev/community-extensions/pull/24